### ISSUE TITLE
Document window allows tabs to be opened, but tabs aren't actually usable

### ISSUE THREAD

**Issue by davidfmiller** [git-up/GitUp#397]
(After opening a new tab, clicking the "File → Open" menu item opens a new document window entirely.)

![screen shot 2017-12-20 at 1 11 05 pm](https://user-images.githubusercontent.com/12614/34228913-43f86fb2-e587-11e7-906d-4714eac4a466.png)

Seems like the quick solution is to remove the "Show Tab Bar" and "Show All Tabs" menu items from the "View" menu.

**Comment by swisspol** [comment]
Tabs are not supported in GitUp. These items should not appear in "View" menu indeed.

**Comment by davidfmiller** [comment]
I'd love to lend a hand on this one. I've forked the GitUp repo and am trying to get it to build, to no avail. I've looked through the source and have got things to the point where the linking fails on `libssh2.a`, `libsqlite3.a`, and `libcrypto.a`.

Trying to find a guide to get things going but haven't come across one in the project wiki or the repo itself... can anyone help this noob out? Thanks.
